### PR TITLE
WIP - DON'T MERGE: Split out sbt-conductr-client

### PR DESCRIPTION
**Pull request by markusjura**
Splitting out the `client` package into a separate library. This is necessary in order to use the control protocol standalone in other projects, e.g. for a ConductR testkit.

This PR also aims to improve the client library in order to fix:
https://github.com/sbt/sbt-conductr/issues/114
https://github.com/sbt/sbt-conductr/issues/83

### CODE REVIEW

**Comment by markusjura** [comment]
@huntc: Can you check if this goes into the right direction. I've splitted up `sbt-conductr` into 2 sub modules:
- client: contains the `ConductRController`, i.e. it sends HTTP requests to the ConductR control server.
- sbt-conductr: Handles the sbt related stuff

Tomorrow I'll fix the issues #114 and #82. 

This PR needs to be merged after @fsat has merged the PR https://github.com/sbt/sbt-conductr/pull/122.

File: sbt-conductr/src/main/resources/LICENSE-2.0.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (huntc) [inline sbt-conductr/src/main/resources/LICENSE-2.0.txt]: Why is this license required here i.e. as part of resources? We have a LICENSE file which is all that is required.

**Comment by huntc** [comment]
The client library feels like something that can potentially have its own release cycle and thus independent of sbt-conductr. For example I can see that it will be used by our test kit. The test kit should have no direct dependency on sbt and could be used outside of an sbt context.

I recommend a separate repository named `conductr-client`.

**Comment by markusjura** [comment]
Duplicate to https://github.com/sbt/sbt-conductr/pull/124

**Pull request closed without merging**